src/components/HTMLEngineProvider/BaseHTMLEngineProvider.js

```diff
@@ -59,6 +59,7 @@ const BaseHTMLEngineProvider = (props) => {
             tagsStyles={styles.webViewStyles.tagStyles}
             enableCSSInlineProcessing={false}
             systemFonts={_.values(fontFamily)}
+            fallbackFonts={{'GTAmericaExp-Regular': fontFamily.EMOJI_TEXT_FONT}}
```

Comment:
Hmm..

Reply:
I like the idea of putting this in a const in this file, and putting the comment right about that